src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -56,6 +60,9 @@ func newBundleCommand(fs afero.Fs) *cobra.Command {
 		Short: "Collect environment data and create a bundle file for the Redpanda Data support team to inspect",
 		Long:  bundleHelpText,
 		Run: func(cmd *cobra.Command, args []string) {
+			path, err := processFilepath(fs, outFile, cmd.Flags().Changed(outputFlag))
+			out.MaybeDie(err, "unable to parse file path %q: %v", outFile, err)
```

Comment:
`unable to determine filepath: %q`, if taking my `determineFilepath` suggestion below